packages/zoe/src/contracts/coveredCall.js

```diff
@@ -1,84 +1,94 @@
 // @ts-check
 
 // Eventually will be importable from '@agoric/zoe-contract-support'
-import {
-  swap,
-  assertIssuerKeywords,
-  assertProposalShape,
-} from '../contractSupport';
+import { assertProposalShape, swapExact } from '../contractSupport';
 
+// Import types.
 import '../../exported';
 
 /**
- * In a covered call, a digital asset's owner sells a call option. A
- * call option is the right to buy the digital asset at a
- * pre-determined price, called the strike price. The call option has
- * an expiry date, when the contract will be cancelled.
+ * A call option is the right (but not the obligation) to buy digital
+ * assets at a pre-determined price, called the strike price. This
+ * call option is "covered," meaning that the owner of the digital
+ * assets must put the assets in escrow. This guarantees that the
+ * assets can be transferred without relying on the owner of the
+ * digital assets to keep their promise.
```

Comment:
It wasn't obvious till later that the call can include multiple assets under different keywords. That's *really* cool and should be more obvious earlier!